src/components/KirtanList/KirtanList.jsx

```diff
@@ -49,8 +38,11 @@ function KirtanList({
     <section className="kirtan-list__container">
       {displayKirtans?.map((kirtan, index) => {
         if (!kirtan.hTitle) kirtan.hTitle = kirtan.Title;
```

Comment:
I believe with new implementation, kirtan.htitle/hSevadar/hAlbum are always available, we can remove this code also...? @isupreetk

Reply:
This makes sense!